ui/raidboss/data/07-dt/raid/r5n.txt

```diff
@@ -1,9 +1,242 @@
+157.6 "--middle--" Ability { id: "A6C5", source: "Dancing Green" }
+164.8 "2-snap Twist" Ability { id: ["A4D5", "A6CC", "A6CD", "A6D0"], source: "Dancing Green" }
+166.3 "2-snap Twist" Ability { id: "A6D1", source: "Dancing Green" }
+168.4 "2-snap Twist (opposite)" Ability { id: "A6D2", source: "Dancing Green" }
+174.0 "Celebrate Good Times" Ability { id: "A6C8", source: "Dancing Green" }
+180.2 "--north--" Ability { id: "A6C5", source: "Dancing Green" }
+185.5 "Ensemble Assemble" Ability { id: "9A30", source: "Dancing Green" }
+192.4 "Arcady Night Fever" Ability { id: "A705", source: "Dancing Green" }
+203.8 "Stone" Ability { id: "98B4", source: "Dancing Green" }
+212.7 "Let's Dance!" #Ability { id: "9BDC", source: "Dancing Green" }
+214.7 "Let's Dance!" #Ability { id: "9BDC", source: "Dancing Green" }
+216.6 "Let's Dance!" #Ability { id: "9BDC", source: "Dancing Green" }
+218.5 "Let's Dance!" #Ability { id: "9BDC", source: "Dancing Green" }
+220.4 "Let's Dance!" #Ability { id: "9BDC", source: "Dancing Green" }
+222.3 "Let's Dance!" #Ability { id: "9BDC", source: "Dancing Green" }
+224.2 "Let's Dance!" #Ability { id: "9BDC", source: "Dancing Green" }
+226.1 "Let's Dance!" #Ability { id: "9BDC", source: "Dancing Green" }
+233.3 "Let's Pose!" Ability { id: "A719", source: "Dancing Green" }
+247.5 "--middle--" Ability { id: "A6C5", source: "Dancing Green" }
+252.8 "Frogtourage" Ability { id: "A704", source: "Dancing Green" }
+267.4 "Moonburn" Ability { id: "A720", source: "Dancing Green" }
+269.4 "Eighth Beats" Ability { id: "A703", source: "Dancing Green" }
+277.7 "Funky Floor" Ability { id: "A6F6", source: "Dancing Green" }
+281.7 "Funky Floor" Ability { id: "A6F6", source: "Dancing Green" }
+284.7 "--middle--" Ability { id: "A6C5", source: "Dancing Green" }
+285.7 "Funky Floor" Ability { id: "A6F6", source: "Dancing Green" }
+289.7 "Funky Floor" Ability { id: "A6F6", source: "Dancing Green" }
+291.8 "4-snap Twist" Ability { id: ["A6DC", "A6DE", "A6E1"], source: "Dancing Green" }
+292.8 "4-snap Twist" Ability { id: "A6E2", source: "Dancing Green" }
+293.2 "4-snap Twist" Ability { id: "A6E3", source: "Dancing Green" }
+293.5 "Funky Floor" Ability { id: "A6F6", source: "Dancing Green" }
+293.5 "4-snap Twist" Ability { id: "A6E4", source: "Dancing Green" }
+295.1 "4-snap Twist (opposite)" Ability { id: "A6E5", source: "Dancing Green" }
+297.5 "Funky Floor" Ability { id: "A6F6", source: "Dancing Green" }
+303.6 "Celebrate Good Times" Ability { id: "A6C8", source: "Dancing Green" }
+309.7 "--middle--" Ability { id: "A6C5", source: "Dancing Green" }
+315.8 "Disco Infernal" Ability { id: "A6F9", source: "Dancing Green" }
+320.9 "Frogtourage" Ability { id: "A704", source: "Dancing Green" }
+335.2 "4-snap Twist" Ability { id: ["A6DC", "A6DE", "A6E1"], source: "Dancing Green" }
+335.4 "Moonburn" Ability { id: "A720", source: "Dancing Green" }
+335.4 "Moonburn" Ability { id: "A71F", source: "Dancing Green" }
```

Comment:
```suggestion
335.4 "Moonburn" Ability { id: ["A71F", "A720"], source: "Dancing Green" }
```

If there's a specific reason to keep these separated, that's fine, but in general I think we should try to keep multiple resolutions of the same name at the same time on a single line.